app/src/main/AndroidManifest.xml

```diff
@@ -1,21 +1,20 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-          package="com.kgmyshin.annict">
+    package="com.kgmyshin.annict">
 
     <application
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/kgmyshin/annict-android/blob/9f7d33eeb56630a8dca6454a13087b7b0625ab7c/app/src/main/AndroidManifest.xml#L5"></span>App is not indexable by Google Search; consider adding at least one Activity with an ACTION-VIEW intent filter. See issue explanation for more details.</td>
    </tr>
  </tbody>
</table>